docs/reference/search/search.asciidoc

```diff
@@ -452,15 +452,36 @@ script has access to the entire context of a document, including the original
 Your script must include `emit` to return calculated values. For
 example:
 +
-[source,js]
+[source,console]
```

Comment:
Quick drive-by:

This does add some of the console chrome to this snippet:
<img width="645" alt="Screen Shot 2021-04-08 at 12 03 36 PM" src="https://user-images.githubusercontent.com/40268737/114059696-a0fbb700-9862-11eb-8593-596cfbbf2871.PNG">

This doesn't mangle the line in copy as cURL, but it's kinda odd... you can't exactly run this in as is in curl or Kibana console. Still probably worth it for the testing coverage tho.

Reply:
Hmmm - I can probably avoid the chrome.

Also, on my screen it fits on the screen. no scroll bar! Hmmm.

Reply:
The scroll bar is likely the result of my window size. 😄

Reply:
+1 on removing that chrome. Let me know how you do it! It could be useful for other work too. For example, I want to switch several of our JS snippets for ingest processors over to console snippets. This would let me just show the object rather than an entire request.

Reply:
![image](https://user-images.githubusercontent.com/215970/114063134-49f7e100-9866-11eb-8f31-8d800767c022.png)

Reply:
@jrodewig I ended up using the `include` trick on a commented out block.That skips the chrome and keeps the test.

Reply:
Thanks @nik9000! Much more readable too!